### PR TITLE
SecuritySystem

### PR DESCRIPTION
**Pull request by jburel**
Update SecuritySystem page

see https://trello.com/c/ZVrR6L3F/91-build-system-changes

### CODE REVIEW

File: omero/developers/Server/SecuritySystem.rst [hunk 1]
```diff
@@ -348,7 +341,7 @@ the server-side :doc:`/developers/Server/Context`. Currently,
 there is no client-side security system. See :ticket:`234`.
 
 The :doc:`/developers/Server/SecuritySystem` and its current only
-implementation BasicSecuritySystem? are somewhat inert and expect
+implementation :server_source:`BasicSecuritySystem <src/main/java/ome/security/basic/BasicSecuritySystem.java>` are somewhat inert and expect
```
Comment (mtbc) [inline omero/developers/Server/SecuritySystem.rst]: Maybe rewrap lines.

**Comment by mtbc** [comment]
Staged at https://merge-ci.openmicroscopy.org/jenkins/job/OMERO-docs/lastSuccessfulBuild/artifact/omero/_build/html/developers/Server/SecuritySystem.html.

File: omero/developers/Server/SecuritySystem.rst [hunk 2]
```diff
@@ -86,21 +86,14 @@ Top-level and build
...
 Client and common
 ~~~~~~~~~~~~~~~~~
 
-the server uses the information in /etc/local.properties to create a Login
-    object. If no Login, Server, or Properties is provided to the
-    ServiceFactory constructor, the empty
-    properties defined in
-    :common_source:`ome/config.xml <src/main/resources/ome/config.xml>`
-    is used.
+The server uses the information in :source:`local.properties <etc/local.properties>`
```
Comment (mtbc) [inline omero/developers/Server/SecuritySystem.rst]: Seems to be a broken link.

**Comment by jburel** [comment]
@mtbc link is fixed now. Ok to merge?

**Review by mtbc — APPROVED** [review]

**Pull request merged**